.travis.yml

```diff
@@ -2,9 +2,23 @@ language: cpp
 compiler:
   - gcc
 before_install:
+  - printenv
+  - export PATH=$PATH:/usr/local/lib
+  - printenv
   - sudo apt-get update -qq
   - sudo apt-get install -qq libboost-all-dev
   - sudo apt-get install maven
+  #### begin cpprest
+  - sudo apt-get install g++ git make zlib1g-dev libboost-all-dev libssl-dev cmake
+  - git clone https://github.com/Microsoft/cpprestsdk.git casablanca
+  - cd casablanca/Release
+  - mkdir build
+  - cd build
+  - cmake .. 
+  - make
+  - sudo make install
+  - cd ../../../
+  #### end cpprest
```

Comment:
I remember that we've been tight on time budget with travis. Do you know how long this takes?

Reply:
upgrading the ubuntu version [https://github.com/travis-ci/travis-ci/issues/6469](https://github.com/travis-ci/travis-ci/issues/6469)

---
In reply to: [187608097](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r187608097) [](ancestors = 187608097)

Reply:
Builds successfully.  Total build time is 37 mins.  We can close this.

---
In reply to: [187651899](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r187651899) [](ancestors = 187651899,187608097)